webstruct/tests/test_evaluation.py

```diff
@@ -0,0 +1,158 @@
+import os
+
+from webstruct import GateLoader, HtmlTokenizer, WebAnnotatorLoader, load_trees
+from webstruct.tests.utils import DATA_PATH
+from webstruct.evaluation import (get_metrics,
+                                  get_metrics_single,
+                                  get_label_entities,
+                                  accuracy,
+                                  precision,
+                                  recall,
+                                  f1_score)
+
+KNOWN_ENTITIES = {'CITY', 'EMAIL', 'ORG', 'STATE', 'STREET', 'SUBJ', 'TEL'}
+EVAL_PATH = os.path.abspath(os.path.join(
+                                DATA_PATH, '..', '..', '..', 'evaluation',
+                                         )
+                            )
+
+
+def almost_equal(result, expected):
```

Comment:
I think https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.allclose.html can cover it